examples/single_column_nutrients_plankton_bacteria_detritus.jl

```diff
@@ -15,8 +15,8 @@ using CairoMakie
 #
 # We set up a single column grid with 4 m grid spacing that's 256 m deep:
 
-grid = RectilinearGrid(size = 64,
-                       z = (-256, 0),
+grid = RectilinearGrid(size = 100,
+                       z = (-1000, 0),
```

Comment:
on the line above it says `grid with 4 m grid spacing that's 256 m deep` so it's good to remedy that to match the grid.